packages/bpk-svgs/tasks/getIconFontMetadataProvider.js

```diff
@@ -61,18 +61,16 @@ export default (codepointsPath: string) => {
         return obj;
       }, {});
 
-    const newFileContents = `${JSON.stringify(sortedCodepoints, null, 2)}\n`;
-
-    return fs.writeFile(absolutePath, newFileContents, err => {
-      const data = err
-        ? null
-        : {
-            path: file,
-            name,
-            unicode: [String.fromCodePoint(codepoints[name])],
-          };
-
-      callback(err, data);
-    });
+    try {
```

Comment:
A little bit of context:

- This logic handles new files (that is, files that are not in the `codepoints.json` file)
- Line 36 handles files that are already present in that file
- The first time `run build` is executed for a new file, it will be handled by this code
- The next times it will be handled in line 36
- Because of how gulp streams work calling the callback asynchronously means it will be "emitted" in a random order (in this case always last as all other files are handled synchronously). 

Handling both situations synchronously ensures we have consistent results between executions.